techniques/failures/F2.html

```diff
@@ -66,15 +48,14 @@ <h3>Using CSS to visually emphasize a phrase or word without conveying that emph
                <p>For styled text that conveys information:</p>
                <ol>
                   <li>Check if there is any styled text that conveys structural information.</li>
-                  <li>Check that in addition to styling, the proper semantic structure is used with the text to convey the information.</li>
+                  <li>Check that in addition to styling, the proper semantic structure is used with the text to convey the visual information.</li>
```

Comment:
```suggestion
                  <li>Check that in addition to styling, the proper semantic structure is used with the text to convey the information.</li>
```

Reply:
Based on discussion with Detlev and prior discussions, pulling back the addition of "visual" as that may be unnecessarily confining what kind of information is conveyed.

Reply:
information/meaning